pythia/tasks/vqa/clevr/dataset.py

```diff
@@ -0,0 +1,121 @@
+import os
+import json
+
+import numpy as np
+import torch
+
+from PIL import Image
+
+from pythia.common.registry import registry
+from pythia.common.sample import Sample
+from pythia.tasks.base_dataset import BaseDataset
+from pythia.utils.general import get_pythia_root
+from pythia.utils.text_utils import VocabFromText, tokenize
+from pythia.utils.distributed_utils import is_main_process, synchronize
+
+
+class CLEVRDataset(BaseDataset):
+    def __init__(self, dataset_type, config, data_folder=None, *args, **kwargs):
```

Comment:
Can we include `dataset_type` and `data_folder ` inside config?

Reply:
To keep the configuration same for train, val, test and avoid repetition, this needs to be added in this way. Though, these implementation choices depend on the developer and can be done differently too.